### PR TITLE
Update title and prose to match CRD living-standard nature

### PR DESCRIPTION
**Pull request by sideshowbarker**
With the switch to autopublishing of the current spec as a CRD at https://www.w3.org/TR/mixed-content/, we have replaced the “Level 1” spec in-place and effectively — and retroactively — made Mixed Content a Living Standard.

As a consequence of that, we need to drop the “Level 2” part from the spec, and drop all references to the TR doc as being the “Level 1” spec that is obsoleted. Instead we need to adjust the spec prose to refer to the current spec as simply an incremental update to itself. Fixes https://github.com/w3c/webappsec-mixed-content/issues/58.

### CODE REVIEW

File: index.bs
Comment (carlosjoan91) [inline index.bs:81]: Maybe switch this to "the core idea of the updated specification is"

**Review by carlosjoan91 — COMMENTED** [review]
For the most part looks good to me, left a comment re: a leftover usage of "Level 2"

File: index.bs
Comment (sideshowbarker) [reply index.bs]: > For the most part looks good to me, left a comment re: a leftover usage of "Level 2"

Thanks for catching that. Please review bad1d13 for some changed language around that.

File: index.bs
Comment (jyasskin) [inline index.bs:81]: While I hate to break a good Spanish Inquisition reference, I think this'll be clearer without it:

```suggestion
  breakage. Instead of advising browsers to simply strictly block all mixed content, this specification advises <i>mixed content autoupgrading</i>:
```

File: index.bs
Comment (jyasskin) [inline index.bs:304]: No need to refactor this in this PR, but since there's no longer a "previous" spec to modify, I think it'll make sense to just incorporate these algorithms into the primary Algorithms section instead of a subsection.

File: index.bs
Comment (jyasskin) [inline index.bs:521]:
```suggestion
  An earlier version of this specification defined the <code>block-all-mixed-content</code> CSP directive. It is now obsolete,
```

And maybe this whole section should be non-normative since we don't need to normatively undefine things that no current specification defines?

**Review by jyasskin — APPROVED** [review]
A couple nits, but looks good overall. Thanks!

File: index.bs
Comment (sideshowbarker) [reply index.bs:303]: > No need to refactor this in this PR, but since there's no longer a "previous" spec to modify, I think it'll make sense to just incorporate these algorithms into the primary Algorithms section instead of a subsection.

I would think so too — but yeah that’s out of scope for this PR at least. I leave it up to @carlosjoan91 to make a call whether and when to actually do that

Comment (carlosjoan91) [reply index.bs:303]: Yeah, incorporating these into algorithms sgtm. I can put a PR together once this one is merged.

File: index.bs
Comment (carlosjoan91) [reply index.bs]: Thanks! The last commit looks good to me.

**Review by carlosjoan91 — APPROVED** [review]
PR looks good to me, thanks

**Pull request merged**
